cmake/modules/AddSwift.cmake

```diff
@@ -401,10 +417,12 @@ function(_add_variant_link_flags)
   elseif("${LFLAGS_SDK}" STREQUAL "ANDROID")
     list(APPEND result
         "-ldl" "-llog" "-latomic" "-licudataswift" "-licui18nswift" "-licuucswift"
+        "--sysroot=${SWIFT_ANDROID_SDK_PATH}"
         "${SWIFT_ANDROID_NDK_PATH}/sources/cxx-stl/llvm-libc++/libs/armeabi-v7a/libc++_shared.so")
-    list(APPEND library_search_directories
-        "${SWIFT_ANDROID_PREBUILT_PATH}/arm-linux-androideabi/lib/armv7-a"
-        "${SWIFT_ANDROID_PREBUILT_PATH}/lib/gcc/arm-linux-androideabi/${SWIFT_ANDROID_NDK_GCC_VERSION}.x")
+    swift_android_lib_for_arch(${LFLAGS_ARCH} ${LFLAGS_ARCH}_LIB)
+    foreach(path IN LISTS ${LFLAGS_ARCH}_LIB)
+      list(APPEND library_search_directories ${path}) 
+    endforeach()
```

Comment:
`swift_target_link_search_directories` may be nicer to use.

Reply:
The function containing these lines does not contain a `target` to call `swift_target_link_search_directories`.

Reply:
Ah, unfortunate.